tests/functional/bundles/ceph.yaml

```diff
@@ -17,19 +23,30 @@ applications:
     - lxd:0
     - lxd:1
     - lxd:2
-  ceph-osd:
+  ceph-osd-hdd:
     charm: cs:~openstack-charmers-next/ceph-osd
     num_units: 3
     options:
       osd-devices: /dev/loop0
     storage:
-      # NOTE (rgildein): The use of a 1.8G size storage device will ensure that it is
-      # not safe to reboot/shutdown two ceph-osd units.
```

Comment:
this comment, does it not apply anymore?

Reply:
That's right.